src/components/Result.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Button, Jumbotron } from 'react-bootstrap';
+
+const Result = ({ score, NumOfQuestions }) => {
+  return (
+    <div>
+      <Jumbotron>
+        <h2>
+          You have got {score} out of {NumOfQuestions}
+        </h2>
+      </Jumbotron>
+      <Button
+        className="mr-2 mb-4"
+        variant="secondary"
+        href="quiz"
+      >
+        Try again
+        </Button>
+      <Button
+        className="mr-2 mb-4"
+        variant="primary"
+      >
+        Review Answers
```

Comment:
```suggestion
        Review answers
```
For all sentences (apart from the title CYF Quiz) we should use sentence case which is where only the first letter is capitalised

Reply:
Yeah, I missed that one.